src/app/rooms/rooms.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { Room } from '../room';
+import { RoomService } from '../room.service';
+
```

Comment:
Add comment explaining what this component is

Reply:
Ok. Done.